### ISSUE TITLE
Accessibility Status of Checkbox in Spelling options not announced when using Screenreader

### ISSUE THREAD

**Issue by peterpacult** [nvaccess/nvda#11693]
### Steps to reproduce:
1. Open Writer
2. Open Spelling check
3. Hit the button for the optons menu
4. Move through the menu with keys

### Actual behavior:
The status of the checkboxes is not announced by screenreader

### Expected behavior:
Screenreader should announce if checkbox is active or not.

#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2020.2

#### Windows version:
Edition	Windows 	10 Home
Version				1909
Operating system build	18363.1016

#### Name and version of other software in use when reproducing the issue:
LibreOffice
Version:	7.0.0.3 (x64)
Build ID:	8061b3e9204bef6b321a21033174034a5e2ea88e
CPU-Threads: 	4; BS: Windows 10.0 Build 18363; UI-Render: Skia/Vulkan; VCL: win
Locale: 	de-DE (de_DE); UI: de-DE
Calc: 	threaded

#### Other information about your system:
---
#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If addons are disabled, is your problem still occuring?
Not tested.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Not tested.

**Comment by leonardder** [comment]
Duplicate of #11690

**Comment by peterpacult** [comment]
This issue has also been reported to the Document Foundation; see https://bugs.documentfoundation.org/show_bug.cgi?id=135920
